boards/arm/rak5010_nrf52840/doc/index.rst

```diff
@@ -6,108 +6,102 @@ RAK5010
 Overview
 ********
 
-The SAM4L series embeds picoPower technology for ultra-low power consumption.
-Combined power control techniques are used to bring active current consumption
-down to 90μA/MHz.  The device allows a wide range of configurations giving the
-user the ability to balance between the lowest possible power consumption and
-the feature set selected for the application.  The WAIT and RETENTION modes
-provide full logic and RAM retention, associated with fast wake-up capability
-(<1.5μs) and a very low consumption of, respectively, 3 μA and 1.5 μA.  In
-addition, WAIT mode supports SleepWalking features.  In BACKUP mode, CPU,
-peripherals and RAM are powered off consuming less than 0.9μA with external
-interrupt wake-up support.
-
-The SAM4L-EK is a full featured design to develop for Atmel SAM4L SoC series.
-The kit is equipped with a rich set of peripherals that make the ATSAM4L-EK a
-perfect evaluation platform.  Download the `SAM4L-EK Online User Guide`_ for
-more details.
-
-.. image:: img/atmel-sam4l-ek-callouts.jpeg
+WisTrio NB-IoT Tracker Pro (RAK5010) is a tracker 
```

Comment:
Please squash all board related commits into one.

Reply:
yes you're right. I thought it would be done by squash/merge directly on Github. I will clean this. thanks for your review